docs/reference/filters.md

```diff
@@ -1528,3 +1528,17 @@ directory, but it won't walk subtrees. For the example case, there
 have to be filenames `write-token` and `read-token` within the
 specified credential paths `/tmp/secrets/`, resulting in
 `/tmp/secrets/write-token` and `/tmp/secrets/read-token`.
+
+## baggageItemToTag
```

Comment:
Is `baggage` a general term used in OpenTracing? Just looking at the filter name `baggageItemToTag` I had no idea it had anything to do with OpenTracing but maybe it's just a convention that I'm not familiar with?

It's documented so I can learn that it's an OpenTracing thing but ideally filter names should be somewhat descriptive on their own.

Reply:
See https://godoc.org/github.com/opentracing/opentracing-go#Span it’s a member of this struct

Reply:
Baggage is indeed a term used in opentracing to represent information valid throughout the process boundaries, unlike Span or Logs https://opentracing.io/docs/overview/tags-logs-baggage/